prosody/Dockerfile

```diff
@@ -1,12 +1,24 @@
 FROM jitsi/base
 
+ADD https://packages.prosody.im/debian/pool/main/p/prosody-trunk/prosody-trunk_1nightly747-1~stretch_amd64.deb /tmp/prosody-trunk.deb
+
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -y lua5.1 lua-expat lua-socket lua-filesystem luarocks gcc libssl1.1 libssl1.0-dev git && \
 	apt-cleanup && \
-	rm -rf /etc/prosody
+	dpkg -i /tmp/prosody-trunk.deb && \
+	rm -rf /etc/prosody /tmp/prosody-trunk.deb
+
+RUN \
+	luarocks install luaossl CRYPTO_LIBDIR=/usr/lib/x86_64-linux-gnu CRYPTO_INCDIR=/usr/include OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu OPENSSL_INCDIR=/usr/include && \
+	luarocks install luajwt CRYPTO_LIBDIR=/usr/lib/x86_64-linux-gnu CRYPTO_INCDIR=/usr/include OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu OPENSSL_INCDIR=/usr/include && \
+	luarocks install luajwtjitsi CRYPTO_LIBDIR=/usr/lib/x86_64-linux-gnu CRYPTO_INCDIR=/usr/include OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu OPENSSL_INCDIR=/usr/include && \
+	luarocks install jwt-jitsi CRYPTO_LIBDIR=/usr/lib/x86_64-linux-gnu CRYPTO_INCDIR=/usr/include OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu OPENSSL_INCDIR=/usr/include && \
+	luarocks install luasec && \
+	luarocks install luabitop
 
 COPY rootfs/ /
+COPY git-modules/jitsi-meet/resources/prosody-plugins /prosody-plugins
```

Comment:
Using a git submodule just for this is quite overkill. Please don't. use ADD for the plugins we specifically want instead. (we don't need all the plugins there)

Reply:
Do you know which plugins we need? I'm sure that we need `token/util.lib.lua` and `mod_token_verification.lua`. Do we need more e.g. `mod_token_auth.lua`?

Reply:
I believe we do, yes.

Reply:
To apply this change I need a full list of all necessary plugins we need.

Reply:
mod_auth_token
mod_token_verification
token/

Reply:
Updated the PR - please have a look.